helm/values.yaml

```diff
@@ -39,6 +39,7 @@ aws:
   # If specified, use the AWS region for AWS API calls
   region: ""
   account_id: ""
+  # Specifying the EKS cluster's endpoint URL here causes issue #1015 - Leave it empty.
```

Comment:
Hi @cparik! Thanks for your contribution! Unfortunately, this (and most) file in the s3-controller repository (and any ACK service controller repository) is generated by the ACK code generator from this template:

https://github.com/aws-controllers-k8s/code-generator/blob/main/templates/helm/values.yaml.tpl#L41

We actually just recently removed the `account_id` option in this section because it was confusing:

https://github.com/aws-controllers-k8s/code-generator/commit/4b30ff5578e2f570d1c5b1741f3098be0d78e246

Feel free to make a patch to the github.com/aws-controllers-k8s/code-generator project instead.

Thank you!
-jay